InfernalMobs/src/main/java/atomicstryker/infernalmobs/common/InfernalMobsCore.java

```diff
@@ -350,36 +319,33 @@ private void loadConfig() {
+            if (isClassAllowed(entity) && (instance.checkEntityClassForced(entity) || entity.getRandom().nextInt(config.getEliteRarity()) == 0)) {
+                try {
                         /*
                             get server world from resource location:
                             RegistryKey<World> registrykey = RegistryKey.create(Registry.WORLD_KEY, resourcelocation);
                             ServerWorld serverworld = p_212592_0_.getSource().getServer().getWorld(registrykey);
                          */
-                        ResourceKey<Level> worldRegistryKey = entity.getCommandSenderWorld().dimension();
-                        ResourceLocation worldResourceLocation = worldRegistryKey.location();
-
-                        // Skip Infernal-Spawn when Dimension is Blacklisted, entries look like: "minecraft:overworld"
-                        if (!config.getDimensionIDBlackList().contains(worldResourceLocation.toString())) {
-                            MobModifier mod = instance.createMobModifiers(entity);
-                            if (mod != null) {
-                                SidedCache.getInfernalMobs(entity.level()).put(entity, mod);
-                                mod.onSpawningCompleteStoreModsAndBuffHealth(entity);
-                                // System.out.println("InfernalMobsCore modded
-                                // mob: "+entity+", id "+entity.getEntityId()+":
-                                // "+mod.getLinkedModName());
-                            }
+                    ResourceKey<Level> worldRegistryKey = entity.getCommandSenderWorld().dimension();
+                    ResourceLocation worldResourceLocation = worldRegistryKey.location();
+
+                    // Skip Infernal-Spawn when Dimension is Blacklisted, entries look like: "minecraft:overworld"
+                    if (!config.getDimensionIDBlackList().contains(worldResourceLocation.toString())) {
+                        MobModifier mod = instance.createMobModifiers(entity);
+                        if (mod != null) {
+                            SidedCache.getInfernalMobs(entity.level()).put(entity, mod);
+                            mod.onSpawningCompleteStoreModsAndBuffHealth(entity);
+                            // System.out.println("InfernalMobsCore modded
+                            // mob: "+entity+", id "+entity.getEntityId()+":
+                            // "+mod.getLinkedModName());
                         }
-                    } catch (Exception e) {
-                        LOGGER.log(org.apache.logging.log4j.Level.ERROR, "processEntitySpawn() threw an exception");
-                        e.printStackTrace();
                     }
-                } else {
-                    setMobWasSpawnedBefore(entity);
+                } catch (Exception e) {
+                    LOGGER.error("processEntitySpawn() threw an exception",e);

```

Comment:
formatting, space before e, which is strange because many changes look like from an automated formatter